website/project/model.py

```diff
@@ -316,6 +352,12 @@ def edit(self, content, auth, save=False):
         self.content = content
         self.modified = True
         self.date_modified = datetime.datetime.utcnow()
+        new_mentions = get_valid_mentioned_users_guids(self, self.node.contributors)
+
+        if new_mentions:
+            project_signals.mention_added.send(self, new_mentions=new_mentions, auth=auth)
+            self.ever_mentioned.extend(new_mentions)
+
         if save:
```

Comment:
The `edit` case should be updated though so that `ever_mentioned` is only modified if `save`.